broker-core/src/main/java/io/zeebe/broker/exporter/stream/ExporterStreamProcessor.java

```diff
@@ -107,16 +107,7 @@ public void onOpen(StreamProcessorContext context) {
 
   @Override
   public long getPositionToRecoveryFrom() {
-    long lowestPosition = -1;
-
-    for (final ExporterContainer container : containers) {
-      container.position = state.getPosition(container.getId());
-
-      if (lowestPosition == -1 || lowestPosition > container.position) {
-        lowestPosition = container.position;
-      }
-    }
-    return lowestPosition;
+    return containers.stream().mapToLong(e -> e.position).min().orElse(-1);
```

Comment:
The position will not contain any value on restart? I think this is the reason why we use the state on the line above

Reply:
What about `containers.stream().mapToLong(c -> state.getPosition(c.getId())).min().orElse(-1)`?